dist/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Leaderboard</title>
+<script defer src="main.js"></script></head>
+<body>
+  <div class="container">
+    <section class="header-section">
+      <h1>Leaderboard</h1>
+  </section>
+  <section class="main-section">
+      <div class="list-section">
+          <div class="list-section-title">
+              <h2>Recent scores</h2>
+              <button type="button" class="btn" id="refresh-btn">Refresh</button>
+          </div>
+          <div class="list">
+            
+            <ul id="leader-lis">
+
+            </ul>
+          </div>
+      </div>
+      <div class="add-section">
+          <div class="section-title">
```

Comment:
- [ OPTIONAL] Kindly Follow our list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) Be careful with blank lines and indentation.
- Do not add blank lines, spaces, or indentations without a reason. For readability:
   - Add blank lines to separate large or logical code blocks.
   - Add two spaces of indentation. Do not use the tab key.

Reply:
Thanks for your feedback, I have fixed this issue